modules/@apostrophecms/translation/index.js

```diff
@@ -27,10 +29,12 @@ module.exports = {
 
           // We don't support multiple providers yet, so just use the first one
           // when no provider is specified.
-          const providerName = self.getProvider(translationProvider)?.name;
-          const module = self.getProviderModule(name);
+          const { name: providerName, module } = self.getProvider(translationProvider);
+          const manager = self.apos.modules[module];
 
-          if (!providerName || !module) {
+          // Might be the responsability of automatic-translation module
+          // since it needs to get the provider and its module anyway
+          if (!providerName || !manager) {
```

Comment:
The error handling should be responsability of the automatic-translation module I think since it has to get the provider and its associated module.